### ISSUE TITLE
Repeater Sub Field Value

### ISSUE THREAD

**Issue by masonry47** [verbb/formie#1255]
### Question

Hi there!  We are wondering about the feasibility of adding the value of a nested field (inside a repeater) to the email subject of the notification.  Logically it makes sense that it is not an available as a variable in the email subject, just by the nature of a repeater field... but in our case, we would want to just get the value of a single field into the email subject, from the first row only... regardless of how many rows are submitted.  Does this seem doable?  Or perhaps there may be some work around? 

Thanks in advance!

**Comment by engram-design** [comment]
Yeah, so Repeater fields (and their sub-fields) aren't available for any tokens, because it's difficult to understand how you'd want to use the values. If you have 3 rows of plain text fields, should they combine their 3 fields' content, or just use the first, second or last repeater row value? Should they be concatenated with commas, or just spaces?

This is illustrated by your example, where you want to grab just the first value. How are we supposed to determine that through mapping, which is simplistic? It's probably not something we'll ever support from a UI level (but you never know)

So for the moment, it's not possible, sorry.

**Comment by engram-design** [comment]
Updated in [4.0.0-beta.4](https://github.com/verbb/formie/releases/tag/4.0.0-beta.4). Repeaters now have support for this sort of functionality.
